Mathlib/Algebra/Lie/Ideal.lean

```diff
@@ -481,6 +481,29 @@ theorem incl_isIdealMorphism : I.incl.IsIdealMorphism := by
   rw [I.incl.isIdealMorphism_def, incl_idealRange]
   exact (I : LieSubalgebra R L).incl_range.symm
 
+variable {I}
+
+theorem comap_bot_le_of_injective (hf : Function.Injective f) : comap f ⊥ ≤ I := by
+  refine le_trans (fun x hx => ?_) bot_le
+  rw [mem_comap, LieSubmodule.mem_bot, ← f.map_zero] at hx
+  exact Eq.symm (hf hx) ▸ Submodule.zero_mem ⊥
+
+theorem comap_bot_of_injective (hf : Function.Injective f) : comap f ⊥ = ⊥ :=
+  le_bot_iff.mp (comap_bot_le_of_injective f hf)
```

Comment:
I'm not convinced about these lemmas since I think we should develop the API for the spelling `f.ker` rather than `comap f ⊥`. Furthermore, a stronger result is true, namely [LieHom.ker_eq_bot](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Lie/Ideal.html#LieHom.ker_eq_bot)
```suggestion
```
For example:
```lean
example (hf : Function.Injective f) : comap f ⊥ = ⊥ := by
  rwa [← f.ker_eq_bot] at hf
```